.github/actions/pr_comment/action.yml

```diff
@@ -0,0 +1,51 @@
+name: Responsible for leaving a comment on a PR
+
+inputs:
+  message:
+    type: string
+    required: true
+  run_once:
+    type: boolean
+    default: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: chromatic_inputs
+      shell: bash
+      run: |
+        echo "message=${{ inputs.message }}"
```

Comment:
Did you intend to leave these in here, or were they for debugging purposes only?

Reply:
So yes, but the name should have been updated (renamed to `pr_comment_inputs`). Here's [an example](https://github.com/JupiterOne/.github/blob/34c55d597a18379cb773738bf9849f7df6a005a8/.github/workflows/test/frontend_runtime_application_pr.test.ts#L54) of how I leverage these simple echo statements in my tests. It basically helps me to map various inputs to the corresponding composite actions, that way you know if a breaking change is made at a higher (i.e. in a workflow) and/or lower (i.e. composite action) level.